app/org/sagebionetworks/bridge/dynamodb/StringIntegerMapMarshaller.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.util.Map;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.JsonMarshaller;
+
+/**
+ * Marshalls Map<String, Integer> values JSON when persisting using DynamoDB.
+ */
+public class StringIntegerMapMarshaller extends JsonMarshaller<Map<String,Integer>> {
+
+}
```

Comment:
You'll need to test this against DynamoDB. That marshaller is there because DDB has a weird thing where it will throw an error if an empty Set is set for that attribute. I don't know what it does with an empty Map... don't know if it can even serialize that by default, I don't know. The DynamoStudyDaoTest runs against DDB so it can be verified to work there with an empty map, and with a map that has a min supported app version mapping.